vividus-plugin-html/src/main/java/org/vividus/util/HtmlUtils.java

```diff
@@ -31,9 +33,19 @@ public static Elements getElements(String html, String cssSelector)
         return getElements("", html, cssSelector);
     }
 
+    public static Elements getElementsByXpath(String html, String xpath)
```

Comment:
does it makes sense to rename css selector methods to make more clear what responsible for what?